test/parallel/test-bootstrap-modules.js

```diff
@@ -43,6 +43,7 @@ const expectedModules = new Set([
   'NativeModule internal/constants',
   'NativeModule internal/encoding',
   'NativeModule internal/errors',
+  'NativeModule internal/event_target',
```

Comment:
FYI added these missing expected modules to `test.parallel/test-bootstrap-modules` which were breaking CI

/cc @addaleax